pom.xml

```diff
@@ -12,19 +12,26 @@
    Header, with the fields enclosed by brackets [] replaced by your own identifying
    information: "Portions copyright [year] [name of copyright owner]".
 
-    Copyright 2015 ForgeRock AS.
```

Comment:
Since these files are imported from their (previously publicly available) source JAR's perhaps we should put the name of the source jar and the md5 and sha-1 hash (both are commonly used with Maven) in the commit message.

Reply:
Done.